CONTRIBUTING.md

```diff
@@ -6,22 +6,22 @@ and
 [GOVERNANCE](https://github.com/spiffe/spiffe/blob/main/GOVERNANCE.md)
 from the SPIFFE project.
 
-# Prerequisites
+## Prerequisites
```

Comment:
I think this was intended to be at the same heading level as "Contributor Guidelines and Governance"

If the linter is forcing us to always have a title heading, then we should probably disable the check or add new title headings so we can avoid upgrading section headings to titles.

Reply:
> or add a new heading line,

I suppose my preference would be to add new title headings ... the rule itself is kind of nice, but I don't feel too strongly